main/src/ca/uwaterloo/flix/language/ast/Kind.scala

```diff
@@ -43,14 +43,19 @@ object Kind {
   case object Star extends Kind
 
   /**
-    * The kind of records.
+    * The kind of records row sets.
```

Comment:
In fact records are not sets of labels they are sequences aka. "rows" :) So I would prefer to keep the existing naming.

Reply:
Gotcha. I had misunderstood some of the wording in the paper.